factcast-store/src/main/java/org/factcast/store/registry/transformation/cache/PgTransformationCache.java

```diff
@@ -57,6 +57,7 @@ public class PgTransformationCache implements TransformationCache, AutoCloseable
   private int bufferThreshold = 1000;
 
   public final int maxBufferSize;
+  private final Object flushLock = new Object();
```

Comment:
```suggestion
```

Reply:
Fixes the found "Maintainability issue" :smile:

Reply:
i had this to do local sync, but as i have the hope it is unnecessay, i removed it again.